interface/wx/toolbar.h

```diff
@@ -271,6 +284,15 @@ class wxToolBarToolBase : public wxObject
         Process a @c wxEVT_TOOL_DROPDOWN event. If unhandled,
         displays the default dropdown menu set using
         wxToolBar::SetDropdownMenu().
+    @event{EVT_TB_CUSTOMIZE(id)}
+        Process a @c wxEVT_TB_CUSTOMIZE event. Under MSW and OSX
+        it will display native customization dialogs. On GTK it will
+        be generic.
```

Comment:
This doesn't seem to be true, there is no implementation at all for MSW and just a stub for GTK. This is not necessarily a problem, but the documentation shouldn't like and you should check that the API you propose is at least in principle implementable under the other platforms.